timApp/static/scripts/tim/plugin/imagex.ts

```diff
@@ -1248,8 +1252,14 @@ interface IAnswerResponse {
             <div class="pluginError" *ngIf="imageLoadError" [textContent]="imageLoadError"></div>
             <h4 *ngIf="header" [innerHtml]="header"></h4>
             <p *ngIf="stem" class="stem" [innerHtml]="stem"></p>
-            <div class="imagex-content">
+            <div class="imagex-content" style="position:relative; height: {{canvasheight}}px">
                 <canvas class="canvas no-popup-menu"
+                        style="position:absolute;"
+                        tabindex="1"
+                        width={{canvaswidth}}
+                        height={{canvasheight}}></canvas>
```

Comment:
Tämä taisi olla ennenkin näin, mutta nyt kun pisti silmään, niin voisi samalla hieman refaktoroida:

```suggestion
                        [width]="canvaswidth"
                        [height]="canvasheight"></canvas>
```

Tässäkin tuo `position: absolute;` voisi ehkä olla `.image-content > canvas`:n tyyli SCSSssä.

Reply:
Korjattu ja siirretty imagex.scss